go/server.go

```diff
@@ -126,6 +127,11 @@ var itemID string
 
 var paymentID string
 
+// The transfer_id is only relevant for Transfer ACH product.
+// We store the transfer_id in memomory - in produciton, store it in a secure
```

Comment:
two typos (make sure to fix these throughout the PR): 

```suggestion
// We store the transfer_id in memory - in production, store it in a secure
```